source/Octopus.Server.Client/Repositories/Async/ReleaseRepository.cs

```diff
@@ -59,6 +59,10 @@ public interface IReleaseRepository : IGet<ReleaseResource>, ICreate<ReleaseReso
         [Obsolete("Please use the overload with cancellation token instead.", false)]
         Task<LifecycleProgressionResource> GetProgression(ReleaseResource release);
         Task<LifecycleProgressionResource> GetProgression(ReleaseResource release, CancellationToken cancellationToken);
+
+        [Obsolete("Please use the overload with cancellation token instead.", false)]
+        Task<GetMissingPackagesForReleaseResponse> GetMissingPackagesForRelease(GetMissingPackagesForReleaseRequest request);
```

Comment:
Why is there an implementation without a cancellation token?

Reply:
I think it was there for backwards compatibility for existing uses, but I thought I'd keep it in there to be consistent even though ours is a new endpoint. Should I remove it?

Reply:
Actually on second thought I think it makes more sense to remove it. I'll do that